README.md

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: MillironX/setup-nextflow@v1
+      - uses: nf-core/setup-nextflow@v1.2.0
```

Comment:
Seems like the `nextflow-io` org would actually be the ideal home for this action?

Reply:
Agreed, not my department though! @ewels